src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar-container">
+      <div class="calendar-container__day-1 calendar-container__day-1--sun"></div>
+      <div class="calendar-container__day-2"></div>
+      <div class="calendar-container__day-3"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-sun calendar--month-length-31">
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
please pay attention to the task
![image](https://user-images.githubusercontent.com/59209337/231087252-514a88d0-3cec-442e-a702-3b0e88627094.png)